### DISCUSSION TITLE
Making image larger on click

### DISCUSSION THREAD

**Discussion by Alternativeheat** [quarto-dev/quarto-cli#10583] [Q&A]
### Description

Hi,

I am currently working on my personal portfolio website.  I have a page with my photographs arranged in 3 rows. What I would like to have is a button(or anything similar) for each image, which on click zooms the image to full screen (or at least make it bigger). 

````qmd
---
title: "Photography"

---

::: {.column-page-inset}

::: {layout-nrow=3 layout-valign="center" }

![Photo 1](/assets/img/photo1.png)
![Photo 2](/assets/img/photo2.png)
![Photo 3](/assets/img/photo3.png)
![Photo 4](/assets/img/photo4.png)
![Photo 5](/assets/img/photo5.png)
![Photo 6](/assets/img/photo6.png)
:::

:::

````

How can I do so? Thanks for your help!

**Comment by mcanouil** [comment]
See lightbox in the documentation.

Also see https://github.com/quarto-dev/quarto-cli/discussions/5972#discussioncomment-10303996
